core/domain/exp_services.py

```diff
@@ -904,8 +832,19 @@ def apply_change_list(
                         change
                     )
                     exploration.update_correctness_feedback_enabled(
-                        edit_correctness_feedback_enabled_cmd.new_value
+                        edit_correctness_feedback_enabled_cmd.new_value)
+                elif change.property_name == 'next_content_id_index':
+                    # Here we use cast because this 'elif'
+                    # condition forces change to have type
+                    # EditExplorationPropertyNextContentIdIndexCmd.
+                    cmd = cast(
+                        exp_domain.EditExplorationPropertyNextContentIdIndexCmd,
+                        change
```

Comment:
Can we use `isinstance`?

Reply:
@sahiljoster32 ?

Reply:
@vojtechjelinek This should not block my PR, right? (As I've followed the existing pattern and even if we decide to use isinstance it won't affect any UX)

Reply:
Yep, sure, not a blocker